pkg/controller/embercsi/embercsi_controller.go

```diff
@@ -303,13 +306,6 @@ func getEnvVars(ecsi *embercsiv1alpha1.EmberCSI, driverMode string) []corev1.Env
 			},
 		)
 	}
-	if len(ecsi.Spec.Config.EnvVars.X_CSI_PERSISTENCE_CONFIG) > 0 {
```

Comment:
-1: Shouldn't we still be able to change the default configuration?  What if the admin doesn't want to use CRDs but use the DB instead?